packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx

```diff
@@ -104,32 +103,28 @@ export const CreateTopicSection = () => {
 
   return (
     <div className="CreateTopicSection">
-      <div className="form">
+      <CWForm
+        validationSchema={topicCreationValidationSchema}
+        onSubmit={handleCreateTopic}
+      >
         <div className="form-inputs">
-          <div>
-            <CWTextInput
-              label="Name (required)"
-              placeholder="Enter a topic name"
-              value={name}
-              onInput={(e) => {
-                setName(e.target.value);
-              }}
-              inputValidationFn={handleInputValidation}
-              autoFocus
-            />
-            {errorMsg && (
-              <CWValidationText
-                className="validation-text"
-                message={errorMsg}
-                status="failure"
-              />
-            )}
-          </div>
-
           <CWTextInput
+            hookToForm
+            label="Name (required)"
+            placeholder="Enter a topic name"
+            name="topicName"
+            value={name}
+            onInput={(e) => {
+              setName(e.target.value);
+            }}
+            inputValidationFn={handleInputValidation}
```

Comment:
since we are using the CWForm, we can get rid of custom validation logic.

Reply:
It would be nice to consolidate all of the validation logic, but I don't think Zod would allow for the complexities in the `handleInputValidation` function.